litmus-portal/graphql-server/pkg/cluster/handler/handler.go

```diff
@@ -144,7 +144,7 @@ func DeleteCluster(clusterID string, r store.StateData) (string, error) {
 		   "apiVersion": "v1",
 		   "kind": "ConfigMap",
 		   "metadata": {
-			  "name": "litmus-portal-config",
```

Comment:
This name may also be stored globally to track it better may be in the configuration file.

Reply:
Done